rules/windows/process_creation/proc_creation_win_reg_add_safeboot.yml

```diff
@@ -0,0 +1,31 @@
+title: Add SafeBoot Keys Via Reg Utility
+id: d7662ff6-9e97-4596-a61d-9839e32dee8d
+related:
+    - id: fc0e89b5-adb0-43c1-b749-c12a10ec37de
+      type: similar
+status: experimental
+description: Detects execution of "reg.exe" commands with the "delete" flag on safe boot registry keys. Often used by attacker to prevent safeboot execution of security products
```

Comment:
' with the "delete" ' but check for copy or add

Reply:
You mean, there should be a "delete" instead of "copy", right?

Reply:
I copied the description from another rule that covers the "delete" and i forgot to change it

Reply:
I did not check , just see the decription and the rule detection did not match.
One is wrong.